main.py

```diff
@@ -43,18 +44,36 @@ def total_dice_roll(total,roll_results, modifier):
     print(roll_results)
     return total
 
-num_dice_input = input(f"How many d6 dice do you want to roll? [1-{MAX_DICE}] ")
-num_dice = check_number_of_dice(num_dice_input)
-
 #Set variables for dice roll
-roll_results = roll_dice(num_dice)
-modifier = 0
 total = 0
+modifier = 0
 
-modifier_input = input("Do you have any modifiers you would like to add? y/n ")
-if modifier_input[0] == "y":
-    modifier = input("How much do you want to add? ")
-    modifier = int(modifier)
+#Ask which character roll they want to use
+check_character_input = input("Which roll do you want to use? ")
+if check_character_input in characters.attributes_skills:
+    skill_input = input("Which skill would you like to use? ")
+    if skill_input in characters.attributes_skills[check_character_input]:
+        num_dice = characters.attributes_skills[check_character_input][skill_input][0]
+        modifier = characters.attributes_skills[check_character_input][skill_input][1]
+    else:
+        print("Sorry skill doesn't exist we will roll the attribute")
```

Comment:
Might be helpful to give the user a menu of numbered options instead of guessing what skills exist